.github/workflows/test.yml

```diff
@@ -293,6 +316,15 @@ jobs:
           rm -rf $temp_dir
           echo "PDFIUM_DYNAMIC_LIB_PATH=$(pwd)/${{ matrix.directory }}/libs/" >> $GITHUB_ENV
 
+      # Sets GOOGLE_APPLICATION_CREDENTIALS in the environment, to be consumed by gcloud or client libraries
+      - name: Generate Google Cloud credential configuration
+        if: matrix.package == '@rust/chonky'
```

Comment:
From the documentation for `403` error codes, there seems to be priority in checking glcoud authentication before checking the valid URL, which implies that for the most part the rest of the workflow is functioning as intended. (link to documentation [here](https://cloud.google.com/vision-ai/docs/error-messages))